wandb/sdk_py27/wandb_artifacts.py

```diff
@@ -430,7 +432,12 @@ def add_reference(
 
         return manifest_entries
 
-    def add(self, obj, name):
+    def add(
+        self,
+        obj,
+        name,
+        _rename_deterministically = False,
```

Comment:
Is there a way to avoid leaking this "_rename_deterministically" thing to the user?

Reply:
removed!